### DISCUSSION TITLE
Any VS Plug-in that can support "find all references" from a proto file to .cs file reference?

### DISCUSSION THREAD

**Discussion by jzabroski** [d1820/proto-attributor#9] [Q&A] [answered]
Hi @d1820 ,

I realize this isn't _your_ plugins particular purpose, but something that I'm just wondering.  Visual Studio supports Find All References for C# langauge (.cs files) but I am not sure something similar exists for Google Proto language ( .proto files).  VS Code has [vscode-proto3](https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3) and [Protobuf (protocol buffers)](https://marketplace.visualstudio.com/items?itemName=pbkit.vscode-pbkit) plug-ins that seem to lap Visual Studio Big Brother.

**Comment by d1820** [comment, marked as answer]
I'm not aware of anything.. if I was going to try and find the matches I
would prob just use a regex search with look ahead matching and groups
keying on the namespace class and property name

This may have been sent with voice to text, so I apologize for any
misspellings or incorrect wording

**Reply by jzabroski** [reply]
Yeah, that's basically what I do now.
